Sources/AccountSDKIOSWeb/Simplified Login/View/SimplifiedLoginViewController.swift

```diff
@@ -188,6 +215,57 @@ class SimplifiedLoginViewController: UIViewController {
         ]
         NSLayoutConstraint.activate(allConstraints)
     }
+    
+    @objc
+    private func didTap(sender: UITapGestureRecognizer) {
+        let location = sender.location(in: containerView)
+        if location.y < 0 {
+            self.dismiss(animated: true, completion: nil)
+        }
+    }
+    
+    @objc
+    private func didPan(sender: UIPanGestureRecognizer) {
+        if sender.state == .ended {
+            let location = sender.location(in: view)
+            if location.y >= 0.7 * UIScreen.main.bounds.height {
+                UIView.animate(withDuration: 0.3) {
+                    self.view.backgroundColor = .clear
+                }
+                self.dismiss(animated: true, completion: nil)
+            } else if let originalTransform = originalTransform {
+                UIView.animate(withDuration: 0.3) {
+                    self.containerView.transform = originalTransform
+                    sender.setTranslation(.zero, in: self.containerView)
+                }
+            }
+        }
+        
+        let translation = sender.translation(in: view)
+        
+        guard translation.y > 0 else {
+            return
+        }
+        
+        containerView.transform = containerView.transform.translatedBy(x: .zero, y: translation.y)
+        sender.setTranslation(.zero, in: containerView)
+    }
+    
+    override var shouldAutorotate: Bool {
+        return true
+    }
+    
+    override open var preferredInterfaceOrientationForPresentation: UIInterfaceOrientation{
+        get {
+            .portrait
+        }
+    }
+    
+    override open var supportedInterfaceOrientations: UIInterfaceOrientationMask{
+        get {
+            .portrait
+        }
```

Comment:
```suggestion
        if isPhone{
            return .portrait
        } else {
            return .all
        }
```

Reply:
The os will call this function on iPad if the app target has [disabled multitasking ](https://stackoverflow.com/a/35275848) and if the viewcontroller is presented full screen (we have it as formSheet though for iPad). 

Might clarify the intent to return .all when device is ipad.